deployment_botpress/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "1"
+services:
+  botpress:
+    image: botpress/server
+    expose:
+      - 3000
+    ports:
+      - "3000:3000"
+    environment:
+      - DATABASE_URL=postgres://postgres:secretpw@postgres:5435/botpress_db
+      # - EXTERNAL_URL=http://localhost/
+    depends_on:
+      - postgres
+    volumes:
+      - ./build/botpress/data:/botpress/data
+
+  postgres:
+    image: postgres:11.2-alpine
+    environment:
+      PGPORT: 5435
+      POSTGRES_DB: botpress_db
+      POSTGRES_PASSWORD: secretpw
+      POSTGRES_USER: postgres
```

Comment:
These will have to be secrets. Can we move them to an .env file? We can have a `template.env` file like the core deployment?

Reply:
Yup, had issues with this but will fix it now